sos/plugins/networking.py

```diff
@@ -17,10 +18,18 @@ class Networking(Plugin):
     plugin_name = "networking"
     profiles = ('network', 'hardware', 'system')
     trace_host = "www.example.com"
-    option_list = [(
+    option_list = [
         ("traceroute", "collect a traceroute to %s" % trace_host, "slow",
-         False)
-    )]
+         False),
+        ("namespaces", "specific namespaces regex to be " +
+         "collected, namespaces should be separated by space " +
+         "as for example \"eth* ens2\"", "fast", ""),
+        ("inspect_namespaces", "define how many namespaces should be " +
```

Comment:
I'm not a huge fan of how the current option names map to their functions.

I think something like `namespace_regex` would make more sense here, with `namespaces` being used for the number of namespaces to iterate over.

Reply:
Parameters renamed as suggested.